requirements.txt

```diff
@@ -2,7 +2,7 @@
 # we want to have control over the version and guarantee that the XML output
 # of our tools is stable
 # NOTE: hard-pinning (==) here gets relaxed to >= in setup.py
-lxml==4.7.1
```

Comment:
I think we want to keep the newer lxml now

Reply:
Oh I missed that during the rebase, just fixed. Thanks for catching!